### PR TITLE
installation script to install flightLGo as systemd service

### PR DESCRIPTION
**Pull request by Ventusfahrer**
includes changes to README.md to reflect the alternativ way to install

### CODE REVIEW

**Comment by snip** [comment]
Thanks for this.
Can you please, instead of changinging the main README, create a installFlightLGoasService.md in the contrib directory with this related content?

Can you please also rename Contrib to contrib?

Thanks

File: README.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ Download binary from https://github.com/snip/flightLGo/releases or build it your
...
+flightlog will output to the terminal it is started in.
+
+```
+Jul 21 13:42:40 mail flightLGo[22695]: Connected to OGN APRS server to track activity of <ICAO> centered on <LAT> <LOGN> with a radius of <RADIUS> km.
+```
+This line indicates, that flightLGo has connected to an APRS Server and registered to get motion events within a circle of <RADIUS> kilometers at the location <LAT> <LONG>. When it receives events it will calcluate whether a takeoff or landing is happening. If so, it will output the event to the terminal which looks like
+```
+Jul 21 14:55:53 mail flightLGo[22695]: 2020-07-21 14:55:46 +0200 CEST> 3EEB98: <D-Callsign> (<CN>) ------------------- Landing
+```
+This event is also sent to the webserver https://logbook.glidernet.org#<ICAO>, where it is stored in a database.
```
Comment (snip) [inline README.md]: Typo in URL?
Should be https://flightlog.glidernet.org/?airfield=<ICAO>
Isn't it?

File: README.md [hunk 2]
```diff
@@ -25,6 +25,23 @@ Download binary from https://github.com/snip/flightLGo/releases or build it your
...
+
+Only takeoff and landing events which are happening while flightLGo is running can be reported to the website.
+The website will only show your airport, if at lease one takeoff or landing event has been watched and reported to it.
+
+If not registered and started as a systemd service or started via a cron job, flightLGo will terminate when you logoff the computer.
```
Comment (snip) [inline README.md]: Not exactly true. I run it without being connected.

File: README.md (hunk 1 above)
Comment (Ventusfahrer) [reply README.md]: well, still not finished. ;)

**Comment by Ventusfahrer** [comment]
hi Seb,

I am done with my changes.

I added the chapter _Check Actiivity_ to README.md, because some questions on OGN google groups did ask for this. Feel free to correct it, leave it as it is or throw it away.

This is the first time I am contributing to a github project. It may not be the best idea to use the online editor for changing the .md files. I had to learn a lot, not only technical stuff (git-specific stuff like fork, clone, pull, push, commit) but also how to contribute my stuff (separate files and directories,..).

Therefore thank you for your patience! I

best regards

Peter 
AKA Ventusfahrer

File: README.md (hunk 2 above)
Comment (Ventusfahrer) [reply README.md]: may be I have to debug it to understand the algorythm. 
You are polling the traffic around the point you have put into the filter?

**Comment by snip** [comment]
A very good start. Thanks a lot.
One more comment: you should have made 2 distinct pull request for adding your stuff + updating existing one as it is 2 independent updates.
But let it like this for this time.
Thanks again.

**Pull request merged**